### PR TITLE
Add Sentiment Analysis

### PR DESCRIPTION
**Pull request by xiamx**
https://github.com/xiamx/awesome-sentiment-analysis

Curated list of Sentiment Analysis methods, implementations and misc. The goal of this repository is to provide adequate links for scholars who want to research in this domain; and at the same time, be sufficiently accessible for developers who want to integrate sentiment analysis into their applications.

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -242,6 +242,7 @@
 - [Speech and Natural Language Processing](https://github.com/edobashira/speech-language-processing)
 	- [Spanish](https://github.com/dav009/awesome-spanish-nlp)
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby)
+        - [Sentiment Analysis](https://github.com/xiamx/awesome-sentiment-analysis)
```
Comment (davisonio) [inline readme.md]: Please add a short description next to this (see other items for example).

**Comment by xiamx** [comment]
Hi @davisonio , addressed your comment, please take another look.

**Comment by sindresorhus** [comment]
> The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

Not followed.

---

`NodeJS` => `Node.js`
`Goland` => `Go`
`CSharp` => `C#`

---

The entries should be in the following format:

```
- Entry - Description.
```

---

Remove `Contents` from the TOC.

---

In `Sentiment Analysis is the field of study that analyzes`, linkify `Sentiment Analysis` to https://en.wikipedia.org/wiki/Sentiment_analysis

---

Would be nice with a illustration: https://www.google.co.th/search?biw=1440&bih=799&tbm=isch&sa=1&btnG=Search&q=sentiment+analysis#tbm=isch&q=%22sentiment+analysis%22

Note:

> includes a project logo/illustration (whenever possible) at the top right of the readme (example). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),

---

The repo should have the following GitHub topics set: awesome-list, awesome, list.

---

Would be very useful with a short summary/description of the papers: https://github.com/xiamx/awesome-sentiment-analysis#survey-papers If you have read any of them.

---

I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community

File: readme.md [hunk 2]
```diff
@@ -242,6 +242,7 @@
 - [Speech and Natural Language Processing](https://github.com/edobashira/speech-language-processing)
 	- [Spanish](https://github.com/dav009/awesome-spanish-nlp)
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby)
+        - [Sentiment Analysis](https://github.com/xiamx/awesome-sentiment-analysis) - Sentiment Analysis for researchers and developers alike.
```
Comment (sindresorhus) [inline readme.md]: Incorrect indentation. Should be one tab.

**Comment by sindresorhus** [comment]
@xiamx Friendly ping :)

**Comment by sindresorhus** [comment]
@xiamx ping2

**Pull request closed without merging**